drivers/include/nrfx_pwm.h

```diff
@@ -94,12 +94,13 @@ typedef struct
     uint8_t output_pins[NRF_PWM_CHANNEL_COUNT]; ///< Pin numbers for individual output channels (optional).
                                                 /**< Use @ref NRFX_PWM_PIN_NOT_USED
                                                  *   if a given output channel is not needed. */
-    uint8_t            irq_priority; ///< Interrupt priority.
-    nrf_pwm_clk_t      base_clock;   ///< Base clock frequency.
-    nrf_pwm_mode_t     count_mode;   ///< Operating mode of the pulse generator counter.
-    uint16_t           top_value;    ///< Value up to which the pulse generator counter counts.
-    nrf_pwm_dec_load_t load_mode;    ///< Mode of loading sequence data from RAM.
-    nrf_pwm_dec_step_t step_mode;    ///< Mode of advancing the active sequence.
+    uint8_t            irq_priority;  ///< Interrupt priority.
+    nrf_pwm_clk_t      base_clock;    ///< Base clock frequency.
+    nrf_pwm_mode_t     count_mode;    ///< Operating mode of the pulse generator counter.
+    uint16_t           top_value;     ///< Value up to which the pulse generator counter counts.
+    nrf_pwm_dec_load_t load_mode;     ///< Mode of loading sequence data from RAM.
+    nrf_pwm_dec_step_t step_mode;     ///< Mode of advancing the active sequence.
+    bool               skip_gpio_cfg; ///< Skip configuration of gpio if it is true.
```

Comment:
I'd rather to rephrase it to shorter version:
```Skip the GPIO configuration.```

Reply:
Well, I think the description should be actually more detailed. Something like:
```
    bool               skip_gpio_cfg; ///< Skip the GPIO configuration.
                                      /**< When this flag is set, the user is responsible for
                                       *   providing the proper configuration of the output pins,
                                       *   as the driver does not touch it at all. */
```

Reply:
I've updated the comments according to your description.

Reply:
I will squash the commits and I've updated the commit message.